src/modules/sub-items/sub.items.module.js

```diff
@@ -244,6 +244,7 @@ export default class SubItemsModule extends Component {
      */
     switchView(activeView) {
         this.setState(() => ({ activeView }));
+        eZ.helpers.tooltips.hideAll();
```

Comment:
I would do it in the `setState` callback as change state can be postponed in react